core/src/db/CMakeLists.txt

```diff
@@ -66,8 +66,8 @@ target_include_directories( milvus_engine
 # **************************** Libraries to be linked with target Engine ****************************
 set( ENGINE_LIBS    pthread
                     libstdc++fs.a
-                    libgomp.a
-                    libgfortran.a
+                    gomp
+                    gfortran
```

Comment:
Keep using static libraries to avoid dependency on the environment at runtime, especially some chaotic system environment.

Reply:
gomp and gfortran are also under LGPL, so the same action should be applied to them.